store/tikv/2pc.go

```diff
@@ -323,17 +324,18 @@ func (c *twoPhaseCommitter) keySize(key []byte) int {
 	return len(key)
 }
 
+func (c *twoPhaseCommitter) skipCheckForWrite() bool {
```

Comment:
Could we add a field in twoPhaseCommitter like SkipConstraintCheck? So we do not need to access GetOption every time.